dpppt-backend-sdk/dpppt-backend-sdk-ws/src/main/java/org/dpppt/backend/sdk/ws/controller/GaenController.java

```diff
@@ -145,14 +148,14 @@ public GaenController(
       @AuthenticationPrincipal
           @Documentation(description = "JWT token that can be verified by the backend server")
           Object principal)
-      throws WrongScopeException, KeyIsNotBase64Exception, DelayedKeyDateIsInvalid {
+      throws DelayedKeyDateIsInvalid, InsertException, WrongScopeException {
     var now = UTCInstant.now();
 
     this.validateRequest.isValid(principal);
 
     // Filter out non valid keys and insert them into the database (c.f. InsertManager and
     // configured Filters in the WSBaseConfig)
-    insertIntoDatabaseIfJWTIsNotFake(gaenRequest.getGaenKeys(), userAgent, principal, now);
+    insertManagerExposed.insertIntoDatabase(gaenRequest.getGaenKeys(), userAgent, principal, now);
 
     this.validationUtils.assertDelayedKeyDate(
```

Comment:
Question: is an `/expose` request valid, even with a missing `delayedKeyDate`?
- yes -> this will fail for these valid requests
- no -> assure that the mobile apps always send a valid `delayedKeyDate`

Reply:
Ok, i just clarified this with the frontend: For v1 there is always as delayedKeyDate. So this is ok.